app/build.gradle.kts

```diff
@@ -73,21 +84,17 @@ android {
 
 dependencies {
     implementation("com.mapbox.navigationux:android:1.0.0-beta.40")
-    implementation("com.mapbox.navigationux:search-api:1.0.0-beta.40")
```

Comment:
Interesting...

Reply:
I think we don't need to explicitly add this module, because it is a part of the standard package